v4/go.mod

```diff
@@ -5,4 +5,5 @@ go 1.13
 require (
 	github.com/newrelic/opentelemetry-exporter-go v0.1.0
 	go.opentelemetry.io/otel v0.10.0
+	google.golang.org/grpc v1.30.0
```

Comment:
What led to this being added? It isn't immediately apparent in the diffs of the other files.

Reply:
This should be from the upgrade of otel to 0.1.0 I believe.